src/proto/tactic.proto

```diff
@@ -16,43 +16,23 @@ message Tactic
     oneof tactic
     {
         AttackerTactic attacker                             = 1;
-        ChipTactic chip                                     = 2;
         CreaseDefenderTactic crease_defender                = 3;
-        DribbleTactic dribble                               = 4;
-        GetBehindBallTactic get_behind_ball                 = 5;
```

Comment:
What are the substitutions to these two tactics (For when we need to get the ball)?

Reply:
Dribble and GetBehindBall are now Skills. You can get a tactic that execute a skill by using the `AssignedSkillTactic` template. 

I haven't added protos for `DribbleSkillTactic` or `GetBehindSkillTactic` that map to `AssignedSkillTactic<DribbleSkill>` and `AssignedSkillTactic<GetBehindBallSkill>` yet. So far I've only added:

https://github.com/UBC-Thunderbots/Software/blob/1a90a987a557f37e8e3835eb28d0c2d56867bfc9/src/proto/tactic.proto#L28-L30

because we need to use them in simulated pytests